tensorrt_llm/_torch/disaggregation/native/bounce/buffer.py

```
@@ -0,0 +1,157 @@
+
+    def close(self) -> None:
+        vm = getattr(self, "_vm", None)
+        if vm is not None:
+            vm.destroy()
+            self._vm = None  # type: ignore[assignment]
+
+    def __del__(self):
+        try:
+            self.close()
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)